### DISCUSSION TITLE
Person who opens issue specify severity

### DISCUSSION THREAD

**Discussion by jeffythedragonslayer** [community/community#19845] [Actions]
Can there be a way for the person who opens an issue to specify what they think is the severity of the issue, like redmine?  Some people do not like waking up to dozens of emails about minor issues all from the same person, and it would be nice if there would be a way to specify the severity and let the maintainer adjust which severity they get emails for.

**Comment by huabin** [comment]
I think this discussion should be the place to discuss issues on GitHub own products, but it has gotten chaos.🤷‍♂️
